client/components/App.jsx

```diff
@@ -29,19 +29,20 @@ const App = () => {
 
           {/* Anxiety */}
         <div className="resources">
-          
-          <Route path = "/"   component = {Depression} />
-          <Route path = "/"   component = {Prevention} />
+            
+          <Route exact path = "/"   component = {Depression} />
+          <Route exact path = "/"   component = {Prevention} />
           <Route exact path = "/young"   component = {Young} />
-          </div>
-          <Link to="/young">Young</Link>
+          <Route exact path = '/register' component={Register} />
+        </div>
+        <Link to="/young">Young</Link>
```

Comment:
please review indentation around this link (line 38) and keep it in relation to the other links above